.github/workflows/copy-workflow.yml

```diff
@@ -12,43 +12,62 @@ jobs:
       matrix:
         cfg: ${{ github.event.client_payload.targets }}
     env:
+      TARGET_REPO_DIR: "target-repo"
```

Comment:
not sure I understand why we need a sub-directory now. maybe document that need?

Reply:
Previously, we used to clone the template repository into a subdirectory of the target directory. Which works fine, until you do a `git add .`. This wasn't a problem until now, because we just deferred cloning the template directory until we had all the commits in place.
With this change, this is not possible any more, since we need the `user-config.json` file from the template directory before we create any commits.